src/components/Item/Item.component.jsx

```diff
@@ -34,7 +62,13 @@ const Item = props => {
             firebase.dataBase
               .collection(localToken)
               .doc(documentId)
-              .update({ lastPurchaseDate: date });
+              .update({
+                lastPurchaseDate: date,
+                numberOfPurchases: numberOfPurchases,
+                nextPurchaseInterval: nextPurchaseInterval,
+                latestInterval: latestInterval,
+                lastEstimate: lastEstimate,
+              });
```

Comment:
Similar suggestion to build the object shorthand as in the pervious comment here:

https://github.com/the-collab-lab/tcl-11-smart-shopping-list/pull/34/files#r486620115